kedro-telemetry/tests/test_plugin.py

```diff
@@ -192,6 +192,8 @@ def test_before_command_run(self, mocker, fake_metadata, caplog):
             "No personal data or IP addresses are stored on our side. "
             "If you want to opt out, set the `KEDRO_DISABLE_TELEMETRY` or `DO_NOT_TRACK` environment variables, "
             "or create a `.telemetry` file in the current working directory with the contents `consent: false`. "
+            "If you don't want to see this message again, "
+            "create a `.telemetry` file in the current working directory with the contents `consent: true`. "
```

Comment:
This all looks great!

but as for this line, I'd almost suggest that the project file is the only place to not allow explicit consent? Since someone could version-control it, and then a user silently gives consent. I'd suggest it has to be an envvar or something, as that's a little harder to confuse for being set :)